src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ComboBoxTests/UnoSamples_Tests.ComboBoxTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.ComboBoxTests
+{
+	[TestFixture]
+	public partial class ComboBoxTests_Tests : SampleControlUITestBase
+	{
+		[Test]
```

Comment:
To run the test on iOS only, use the `ActivePlatform` attribute: 
https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#selectively-ignore-tests-per-platform

(It's been introduced after you submitted your PR)